### PR TITLE
Cleans up settings.py

### PR DESCRIPTION
**Pull request by KasparPeterson**

### CODE REVIEW

File: opencopilot/application.py [hunk 1]
```diff
@@ -41,23 +42,22 @@ def __init__(
         openai_api_key: Optional[str] = None,
         copilot_name: str = "default",
         host: str = "127.0.0.1",
-        api_base_url: str = "http://127.0.0.1/",
         api_port: int = 3000,
         environment: str = "local",
         allowed_origins: str = "*",
-        weaviate_url: Optional[str] = None,
+        weaviate_url: Optional[str] = "http://localhost:8080",
```
Comment (kgrofelnik) [inline opencopilot/application.py:48]: Is this correct?
If set to None, we use embedded Weaviate, which should be the default behavior.

File: opencopilot/application.py [hunk 2]
```diff
@@ -41,23 +42,22 @@ def __init__(
...
         max_document_size_mb: int = 50,
-        slack_webhook: str = "",
         auth_type: Optional[str] = None,
-        api_key: str = "",
-        jwt_client_id: str = "",
-        jwt_client_secret: str = "",
-        jwt_token_expiration_seconds: int = timedelta(days=1).total_seconds(),
-        helicone_api_key: str = "",
-        helicone_rate_limit_policy: str = "3;w=60;s=user",
+        api_key: Optional[str] = None,
```
Comment (taivop) [inline opencopilot/application.py:54]: I never understood what this is, perhaps should rename? or at least document well

Comment (KasparPeterson) [reply opencopilot/application.py:54]: resolved face 2 face

**Review by taivop — APPROVED** [review]
approved based on reading but not testing

**Pull request merged**